static/js/controllers/configuration.js

```diff
@@ -41,6 +41,14 @@ angular.module('inboxControllers').controller('ConfigurationCtrl',
         }
       },
       {
+        state: 'configuration.upgrade',
+        icon: 'fa-rocket',
```

Comment:
Not the right icon but I can't find a good one. Maybe: http://fontawesome.io/icon/cloud-upload/ ?